develop/devguide/Automation/UIBlockTypesOverview.md

```diff
@@ -459,7 +459,7 @@ uiBuilder.AppendBlock(blockItem);
 > - The `AutomationTimeUpDownOptions` property `AllowSpin` is not supported in the Low-Code Apps.
 > - The `AutomationTimeUpDownOptions` property `UpdateValueOnEnter` is not supported in Cube.
 > - The `AutomationTimeUpDownOptions` property `FractionalSecondsDigitsCount` is only supported in Cube and should be within a range of 0 to 3.
-> - From DataMiner 10.3.0 [CU1]/10.3.4 onwards, the `ShowTimeUnits` property is available. When this property is set to *true*, the component will display labels indicating the days, hours, minutes and seconds. By default, this property is set to *false*.
+> - From DataMiner 10.3.0 [CU1]/10.3.4 onwards, the `ShowTimeUnits` property is available. When this property is set to *true*, the component will display labels indicating the days, hours, minutes and seconds. By default, this property is set to *false*. The `ShowTimeUnits` property is only supported in the DataMiner web apps and not in DataMiner Cube.
```

Comment:
Is this OK? @lander-vanhaverbeke-skyline

Reply:
Ok!